### PR TITLE
Add activity checkbox to procedure 

### PR DESCRIPTION
**Pull request by A1Gard**
The added procedure was dedicative by default in other versions not in latest.
But now the procedures added by default is active and not any direct way for change this by admin.
I put an checkbox for chanage activity.

### CODE REVIEW

File: interface/orders/types_edit.php [hunk 1]
```diff
@@ -366,6 +367,17 @@ class='inputtext' />
     style='width:100%' readonly />
   </td>
  </tr>
+ <tr>
+     <td nowrap> &nbsp;</td>
+  <td>
+      <label> 
+        <?php echo xlt('Activity'); ?>
+          <input type='checkbox'  name='form_activity' value='1' <?php if ($row['activity']) {
+              echo 'checked=""' ;
```
Comment (sunsetsystems) [inline interface/orders/types_edit.php]: For a new procedure type you want this to default to checked. Doesn't look like this will do that.

**Comment by sunsetsystems** [comment]
This seems like a reasonable thing to do, but are you aware of anything (other than my recent fix) that actually checks this flag? Or are you planning something that does?

**Comment by A1Gard** [comment]
@sunsetsystems We assume any user want disable a procedure, How user can disable a procedure? it's not possible now! but with this commit user can disable the procedure.

File: interface/orders/types_edit.php (hunk 1 above)
Comment (A1Gard) [inline interface/orders/types_edit.php]: I'll fix this and set default checked tomorrow .

**Comment by sunsetsystems** [comment]
OK but just want to note that (as best I can tell) a disabled procedure will still be available for selection in the procedure order form. So it seems there is more work to do in support of this feature. Of course that can come later.

**Comment by A1Gard** [comment]
@sunsetsystems Ok dear I got it. Now my pull request is benefit or not?
I can update the procedure search form to as openemr team need too.
Please give me a feedback.

**Comment by sunsetsystems** [comment]
Your PR is OK. You could update the procedure search in this one, or create a new one for that. Also suggest you check other places that the procedure_type table is queried, and see if the activity flag should be checked in those as well. I appreciate your work on this!

**Comment by robertdown** [comment]
Closing this since outdated PR

**Pull request closed without merging**